app/validators/cross_model_slug_validator.rb

```diff
@@ -14,21 +14,21 @@ def validate_each(record, attribute, value)
   private
 
   def not_on_reserved_list?(record, attribute, value)
-    return unless ReservedWords.all.include?(value)
+    return false unless record.class.name != "Page" && ReservedWords.all.include?(value)
```

Comment:
`unless` + `!=` makes the condition a bit harder to understand, I would change to `if`, smth like:

`if record.is_a?(Page) || !ReservedWords.all.include?(value)`